From 9880ded642b3cdea2aac59b0cdfcecf8317502cf Mon Sep 17 00:00:00 2001
From: Eero Tamminen <oak@helsinkinet.fi>
Date: Thu, 5 Mar 2020 21:40:22 +0200
Subject: [PATCH] incompatible.txt update

Christian Zietz debugged Cubase v2 crash and verified that it happens
for the same reason as Cubase Lite crash.  He also found out that
after patching that issue, there are further crash bugs in Cubase
(i.e. creating a Cubase loader/TSR for EmuTOS it wouldn't be very
simple).

Moved "Bug analysis:" in next program to its own line for consistency.
---
 doc/incompatible.txt | 27 ++++++++++++++-------------
 1 file changed, 14 insertions(+), 13 deletions(-)

diff --git a/doc/incompatible.txt b/doc/incompatible.txt
index 47b2ba47..bfec03b7 100644
--- a/doc/incompatible.txt
+++ b/doc/incompatible.txt
@@ -178,28 +178,29 @@ Use a less-buggy control panel, e.g. XCONTROL.ACC from the TT or
 Falcon Language Disk.
 
 
-Program: Cubase Lite
---------------------
+Program: Cubase v2 & Lite
+-------------------------
 Error: panic
 
 Bug analysis:
-On TOS 1.62 etv_timer vector is a delegate to an internal private
-function.  Cubase Lite tries to guess the address of that private
-function in an undocumented way, which crashes on EmuTOS.  (Somebody
-could write a loader or TSR to change the etv_timer function so that
-Cubase will not crash.)
+On TOS (v1.62) etv_timer vector is a delegate to an internal private
+function.  Cubase tries to guess the address of that private function
+in an undocumented way, which crashes on EmuTOS. If that is patched,
+crash happens on next private function which Cubase tries to search
+for.
 
 
 Program: Luftschloss
 --------------------
 Error: Fails to load with an error message about the MPB variable.
 
-Bug analysis: Luftschloss is a reset resident RAM disk originally
-distributed with the German book "Scheibenkleister". It does its
-memory management itself, without using GEMDOS functions. For that,
-it needs to determine the location of an undocumented system variable,
-the memory parameter block (MPB). The heuristic it uses to find the
-memory address of that variable only works with Atari TOS.
+Bug analysis:
+Luftschloss is a reset resident RAM disk originally distributed with
+the German book "Scheibenkleister". It does its memory management
+itself, without using GEMDOS functions. For that, it needs to
+determine the location of an undocumented system variable, the memory
+parameter block (MPB). The heuristic it uses to find the memory
+address of that variable only works with Atari TOS.
 
 
 Program: OMIKRON.BASIC V3.01 (interpreter)
-- 
2.20.1

